static/test_io_close.py

```
@@ -0,0 +1,22 @@
+# _*_ coding:utf-8 _*_
+# !/usr/bin/env python
+
+import html, io, json, os, pickle, random, re, sqlite3, string, sys, subprocess, time, traceback, urllib.parse, \
+    urllib.request, xml.etree.ElementTree
+import lxml.etree
+
+
+def func1():
+    # 检测出
+    fd = open('test.txt')
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)